store/mockstore/mocktikv/cluster.go

```diff
@@ -545,6 +545,7 @@ func (c *Cluster) getRegionsCoverRange(start, end MvccKey) []*Region {
 
 // Region is the Region meta data.
 type Region struct {
+	rw     sync.RWMutex
```

Comment:
The `Cluster` which is the caller of `Region` already has a Lock, could it be unified with this Lock?